.changeset/stupid-poets-grab.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/markdoc': minor
```

Comment:
This should be safe as a `patch` change actually! Seems to follow the semver rules for a pre-1.0 package:
- ✅  patch - fixes _and_ nonbreaking features
- minor - breaking features